### ISSUE TITLE
Can someone make a simple example of how to use it?

### ISSUE THREAD

**Issue by JuanDa237** [Alexandre-Fernandez/astro-i18n#82]
It can be interesting a [stackblicks](https://stackblitz.com/) or a simple repo with a example of how to use it.

**Comment by JuanDa237** [comment]
i installed it, run the command for the middleware, the command for generate the pages, i am using the {t("your_common")} for test if it works, but i am unable of make it work

**Comment by JuanDa237** [comment]
i run the example on apps/astro and it dont work

**Comment by irg1008** [comment]
Yeah it just displays the key not the json value. We could say this is absolutely broken then. It's the main feature

**Comment by irg1008** [comment]
It works using config translations object but it doesn't with file system based jsons, not in i18n directory nor pages directory

**Comment by rienheuver** [comment]
I can confirm this, have been messing with the config for a while now in all sorts of combinations, but the only one I can get to work is in the config file itself
